wiki/Tournaments/GBC/GBC_2024_Autumn/en.md

```diff
@@ -0,0 +1,280 @@
+5. After each pick, both captains **may** ban **one** of the opposing team members from playing the beatmap in a match, **except during tiebreakers**.
+   - Any player can only be banned like this **once** per match.
+   - Only players who have already played in the match can be banned.
+   - The team that picks beatmaps first cannot ban players during the first 2 rounds.
+     - As a consequence of the earlier rules, no teams are allowed to ban players during the first round since no one has finished playing any beatmaps.
+   - Picks and bans should be sent in the same message. If the ban is delayed, or the message causes a disagreement or misunderstanding, the referee has the right to ignore the ban.
+6. Teams will have 2 minutes to pick a beatmap and ban an opposing player, and 2 minutes to get ready.
+   - A team can request a timeout when they are picking a map. The timeout lasts 2 minutes and will be used after the map pick timer.
+   - No matter which team needs to timeout after their pick, it can only allowed with the consent of both captains.
+   - The tactical timeout may be called on a tiebreaker.
+7. If a team takes more time than allowed, the procedures adopted will be as follows:
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee.
+   - On subsequent occurrences:
+     - For bans of opposing players, the captain forfeits the ban.
+     - For map bans, the captain forfeits the ban.
+     - For map picks, a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the tiebreaker.
+     - For a ready timer, the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+   - Repeat offenders may receive further sanctions from the host.
+
+### Mappool instructions
+
+1. There will be six separate mappools.
+2. Each mappool consists of a fixed amount of maps each stage which will all be played under [FreeMod](/wiki/Gameplay/Game_modifier#free-mod) conditions. This means that there is a unique FreeMod bracket.
+3. The mappool sizes are as follows:
+   - Qualifiers: 8 beatmaps
+   - Mappool in the regular phase: 15 beatmaps
+   - Mappool in the elimination phase: 17 beatmaps.
+4. Each mappool has one tiebreaker, except for the Qualifiers. The tiebreaker will be played under FreeMod conditions.
+5. Possible mod choices for the FreeMod bracket are `NoFail`, `Hidden`, `Fade In`, `Flashlight`, and `Mirror`.
+6. The round 2 & round 3 in a Swiss Round will use same mappools, the final week will use same mappool.
```

Comment:
please check if this is correct

```suggestion
6. Round 2 and round 3 in the Swiss rounds will use same mappools, while the final week will use a different mappool.
```

Reply:
no,
- we have 2 Swiss, and for both of them, r2 and r3 will use same pool.
- final week will might have 2 rounds, but use same mappool.

Reply:
so 

```suggestion
6. Round 2 and 3 in the Swiss rounds, as well as the final week, will use same mappool.
```

Reply:
as in there is **one** mappool used by everything?

Reply:
![8a5fbcbb2fd3541396c2600a5067f56e](https://github.com/user-attachments/assets/b2b454c3-ede6-44b6-9244-f05603553024)

Reply:
```suggestion
6. In both Swiss rounds, weeks 2 and 3 will use the same mappool.
```

where is the "final week" in that image?

Reply:
recommend `Round 2 and 3 in a Swiss round will use the same mappool, and each knockout week will use a different mappool`

Reply:
alternatively

```suggestion
6. Mappools for each week are structured as follows:

| Stage | Mappool |
| :-- | :-- |
| Swiss I week 1 | A |
| Swiss I week 2 & 3 | B |
| Swiss II week 1 | C |
| Swiss II week 2 & 3 | D |
| Knockout week 1 | E |
| Knockout week 2 | F |
```